packages/common/src/services/pagination.service.ts

```diff
@@ -437,6 +492,10 @@ export class PaginationService {
     }
   }
 
+  updatePageInfo(pageInfo: PageInfo) {
```

Comment:
it doesn't update much, it might be better as `setPageInfo` since it's a simple assignment